packages/react-ui/storybook/stories/Introduction.stories.mdx

```diff
@@ -11,12 +11,27 @@ import Storybook from '../assets/storybook.svg';
 # CARTO for React - Widgets and UI Components
 
 This storybook includes the UI Components (@carto/react-ui) for CARTO for React.
-They are based on [Mui](https://mui.com/), and they include a CARTO theme + custom widgets.
+
+## Tech stack
+
+The components are based on [Mui v5](https://mui.com/), and they include a CARTO theme + custom components and widgets.
+
+We have support for `React` 17 & 18.
 
 ## Library version
 
 <p className='text-tag'>{version}</p>
 
+## Development state
+
+We have a label in the story of each component that indicates its maturity and recommended use:
+
+- `Deprecated`: Legacy component that might be currently in use, but shouldn`t be used in new projects
+- `In development`: Work in progress or experimental. Not ready to use in production yet
+- `Ready to review`: Ready to review and validation by CARTO design team and other stakeholders, prior to be consider fully done
+- `Validated`: Fully validated and ready-to-use implementation
+- `Needs update`: Needs an update due to some design changes, but can be used in this state
```

Comment:
I would suggest something like this:

```suggestion
**Recommended use**
- `Validated`: Fully validated and ready-to-use implementation of a component
- `Needs update`: Needs an update due to some design changes, but it can be used in this state
- `Deprecated`: Legacy component that might be currently in use, but it shouldn't be used in new projects

**Not supported**
- `In development`: Work in progress or experimental. Not ready to use in production yet
- `Ready to review`: Ready to review and validation by CARTO design team and other stakeholders, prior to be consider fully done (it can be considered as a 2nd phase of `In development`).

```